pinot-core/src/main/java/org/apache/pinot/core/data/function/DefaultFunctionEvaluator.java

```diff
@@ -47,11 +47,13 @@
 public class DefaultFunctionEvaluator implements FunctionEvaluator {
   // Root of the execution tree
   private final ExecutableNode _rootNode;
+  private final DefaultFunctionRegistry _defaultFunctionRegistry;
```

Comment:
The name "Default" started becoming confusing and unintuitive to me. How about we rename Default to Inbuilt?
So we'll have InbuiltFunctionEvaluator and InbuiltFunctionRegistry. wdyt? I feel it's more clear, and anyway we refer to them as inbuilt functions when talking about them to someone.
You can also rename tests (pretty much whatever intellij makes you rename when renaming these classes)